FluentTerminal.App/Strings/es/Resources.resw

```diff
@@ -745,7 +745,7 @@
     <value>Cambiar ventana</value>
     <comment>Command</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```

Comment:
this has been renamed to NewSshTab with #410

Reply:
The same answer as above...

Reply:
Fixed.